numero.Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM golang:1.17.6-alpine3.15 AS dependencies
+
+RUN apk update && apk add --no-cache \
+  ca-certificates \
+  musl-dev \
+  gcc \
+  build-base \
+  git
+
+ENV GO111MODULE=on \
+  CGO_ENABLED=1 \
+  GOOS=linux \
+  GOARCH=amd64
+
+WORKDIR /build
+
+COPY go.mod go.sum ./
+RUN go mod download
+
+FROM dependencies AS builder
+
+COPY . .
+
+RUN ./build.sh
+
+WORKDIR /dist
+
+RUN cp /build/main .
+
+FROM alpine:3.15 as dist
+COPY custom_account_setup.cdc ./
+COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+COPY --from=builder /dist/main /
+
+CMD FLOW_WALLET_PORT=$PORT FLOW_WALLET_DATABASE_DSN=$DATABASE_URL FLOW_WALLET_IDEMPOTENCY_MIDDLEWARE_REDIS_URL=$REDIS_URL /main
```

Comment:
We need to pass these env vars in here because they need to be set by Heroku-generated values.

A note about Redis - it is not required that we use redis for this setup. I  believe it is just being used as the [ Idempotency middleware](https://github.com/numeroai/flow-wallet-api), and it's possible to use the postgres db for this instead. But based on the README it is recommended to use redis for this purpose in production.

Reply:
Got it. For this, do we need to then add a redis instance to the deployed environments on Heroku?